### PR TITLE
Addition of custom user agent.

### PR DESCRIPTION
**Pull request by rhallick**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by rhallick** [comment]
Hi there - just wondering if I need to do anything more in order to get this progressed please?  Thank you.

File: Packs/FeedIntel471/Integrations/Intel471Malware/Intel471Malware.py [hunk 1]
```diff
@@ -47,6 +47,11 @@
 INDICATOR_VALUE_FIELD = {FeedIndicatorType.File: 'indicator_data_file_sha256',
                          FeedIndicatorType.URL: 'indicator_data_url',
                          "ipv4": 'indicator_data_address'}
+DEMISTO_VERSION = demisto.demistoVersion()
+CONTENT_PACK = 'Intel471 Feed/2.0.1'
+INTEGRATION = 'Intel471 Malware Feed'
+USER_AGENT = 'XSOAR/' + DEMISTO_VERSION['version'] + '.' + DEMISTO_VERSION['buildNumber'] \
+    + ' - ' + CONTENT_PACK + ' - ' + INTEGRATION
```
Comment (Bargenish) [inline Packs/FeedIntel471/Integrations/Intel471Malware/Intel471Malware.py:54]:
```suggestion
USER_AGENT = f'XSOAR/{DEMISTO_VERSION["version"]}.{DEMISTO_VERSION["buildNumber"]} - {CONTENT_PACK} - {INTEGRATION}'
```

File: Packs/FeedIntel471/Integrations/Intel471MalwareIndicator/Intel471MalwareIndicator.py [hunk 2]
```diff
@@ -50,6 +50,11 @@
 INDICATOR_VALUE_FIELD = {FeedIndicatorType.File: 'indicator_data_file_sha256',
                          FeedIndicatorType.URL: 'indicator_data_url',
                          "ipv4": 'indicator_data_address'}
+DEMISTO_VERSION = demisto.demistoVersion()
+CONTENT_PACK = 'Intel471 Feed/2.0.1'
+INTEGRATION = 'Intel471 Malware Indicator Feed'
+USER_AGENT = 'XSOAR/' + DEMISTO_VERSION['version'] + '.' + DEMISTO_VERSION['buildNumber'] \
+    + ' - ' + CONTENT_PACK + ' - ' + INTEGRATION
```
Comment (Bargenish) [inline Packs/FeedIntel471/Integrations/Intel471MalwareIndicator/Intel471MalwareIndicator.py:57]:
```suggestion
USER_AGENT = f'XSOAR/{DEMISTO_VERSION["version"]}.{DEMISTO_VERSION["buildNumber"]} - {CONTENT_PACK} - {INTEGRATION}'
```

File: Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.py [hunk 3]
```diff
@@ -20,6 +20,11 @@
     'links_reportTotalCount': 'intel471reporttotalcount',
     'links_instantMessageTotalCount': 'intel471instantmessagetotalcount',
 }
+DEMISTO_VERSION = demisto.demistoVersion()
+CONTENT_PACK = 'Intel471 Feed/2.0.1'
+INTEGRATION = 'Intel471 Actors Feed'
+USER_AGENT = 'XSOAR/' + DEMISTO_VERSION['version'] + '.' + DEMISTO_VERSION['buildNumber'] \
+    + ' - ' + CONTENT_PACK + ' - ' + INTEGRATION
```
Comment (Bargenish) [inline Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.py:27]:
```suggestion
USER_AGENT = f'XSOAR/{DEMISTO_VERSION["version"]}.{DEMISTO_VERSION["buildNumber"]} - {CONTENT_PACK} - {INTEGRATION}'
```

File: Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.py [hunk 4]
```diff
@@ -53,14 +58,15 @@ def custom_build_iterator(client: Client, feed: Dict, limit: int = 0, **kwargs)
     result: List[Dict] = []
     should_continue = True
     total_count = 0
+    request_headers = {'user-agent': USER_AGENT}
```
Comment (Bargenish) [inline Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.py:61]: Let's remove this line and define the `headers` as a parameter in the main function  and pass it to the feed_main function instead.
And in the main function #L166
```python
params['headers'] = {'user-agent': USER_AGENT}

```
_**Relevant for all the feeds**_

File: Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.py [hunk 5]
```diff
@@ -53,14 +58,15 @@ def custom_build_iterator(client: Client, feed: Dict, limit: int = 0, **kwargs)
...
 
     while should_continue:
         r = requests.get(
             url=url,
             verify=client.verify,
             auth=client.auth,
             cert=client.cert,
-            headers=client.headers,
+            headers=request_headers,
```
Comment (Bargenish) [inline Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.py:69]:
```suggestion
            headers=client.headers,
```

**Review by Bargenish — CHANGES_REQUESTED** [review]
@rhallick Thank you for you contribution.

Please see my notes below, I gave an example to the change that need to get done in all the feeds in the `Packs/FeedIntel471/Integrations/Intel471Actors/Intel471Actors.py` file.

**Review by Bargenish — APPROVED** [review]

**Pull request merged**